storage/src/index/storage.rs

```diff
@@ -251,7 +243,7 @@ impl<'a, T: Translator, V: PartialEq + Eq> Cursor<'a, T, V> {
 
     /// Deletes the current value, adjusting the list structure.
     ///
-    /// Increments the `pruned` counter to track removals.
+    /// Increments the `pruned` counter and  decrements the `items` counter.
```

Comment:
nit: `items` gauge.

Reply:
Actually, let's just remove that line.

Reply:
The function comment shouldn't tell you how the metrics work inside lol.

Reply:
removed!